service/administration/userGroup.go

```diff
@@ -0,0 +1,305 @@
+package administration
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/jsonutil"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/uritemplates"
+)
+
+type UserGroup struct {
+	Description *string            `json:"description,omitempty"`
+	Name        *string            `json:"name,omitempty"`
+	Policies    []*UserGroupPolicy `json:"policies,omitempty"`
+	UserIds     []string           `json:"userIds,omitempty"`
+	UserGroupId *string            `json:"id,omitempty"`
+	Users       []*UserGroupUser   `json:"users,omitempty"`
+	CreatedAt   *string            `json:"createdAt,omitempty"`
+	PolicyNames []string           `json:"policyNames,omitempty"`
+	UsersCount  *int               `json:"usersCount,omitempty"`
+
+	// forceSendFields is a list of field names (e.g. "Keys") to
+	// unconditionally include in API requests. By default, fields with
+	// empty values are omitted from API requests. However, any non-pointer,
+	// non-interface field appearing in ForceSendFields will be sent to the
+	// server regardless of whether the field is empty or not. This may be
+	// used to include empty fields in Patch requests.
+	forceSendFields []string
+
+	// nullFields is a list of field names (e.g. "Keys") to include in API
+	// requests with the JSON null value. By default, fields with empty
+	// values are omitted from API requests. However, any field with an
+	// empty value appearing in NullFields will be sent to the server as
+	// null. It is an error if a field in this list has a non-empty value.
+	// This may be used to include null fields in Patch requests.
+	nullFields []string
+}
+
+type UserGroupPolicy struct {
+	AccountIds []string `json:"accountIds,omitempty"`
```

Comment:
get user group details has this as account names, check in postman once

Reply:
It is incorrect in API spec, the response is retrurning account Ids not names